lib/devicetrust/native/api.go

```diff
@@ -46,3 +48,22 @@ func GetDeviceCredential() (*devicepb.DeviceCredential, error) {
 func SolveTPMEnrollChallenge(challenge *devicepb.TPMEnrollChallenge) (*devicepb.TPMEnrollChallengeResponse, error) {
 	return solveTPMEnrollChallenge(challenge)
 }
+
+// SolveTPMAuthDeviceChallenge completes a TPM device authetication challenge.
+func SolveTPMAuthDeviceChallenge(challenge *devicepb.TPMAuthenticateDeviceChallenge) (*devicepb.TPMAuthenticateDeviceChallengeResponse, error) {
```

Comment:
Maybe:

```suggestion
func SolveTPMAuthnDeviceChallenge(challenge *devicepb.TPMAuthenticateDeviceChallenge) (*devicepb.TPMAuthenticateDeviceChallengeResponse, error) {
```

and similarly throughout?

Reply:
Yeah nice catch

Reply:
8035075682